src/bin/searchLogs.ts

```diff
@@ -0,0 +1,50 @@
+import nano from 'nano'
+
+import { config } from '../config'
+
+const nanoDb = nano(config.couchDbFullpath)
+const { searchTerm } = config
+
+main().catch(e => {
+  console.error(e.message)
```

Comment:
It's typical to do `process.exit(1)`, so the terminal will see that it failed too.